src/ServiceInsight/Saga/SagaResources.xaml

```diff
@@ -11,6 +13,26 @@
         <Setter Property="Background" Value="#F8F8F8" />
         <Setter Property="Foreground" Value="#777777" />
     </Style>
+    
+    <Style x:Key="LinkText" TargetType="{x:Type TextBlock}">
+        <Setter Property="Foreground" Value="{StaticResource SagaBlue}" />
+        <Setter Property="Cursor" Value="Hand" />
+        <Style.Triggers>
+            <Trigger Property="IsMouseOver" Value="True">
+                <Setter Property="TextDecorations" Value="Underline" />
+            </Trigger>
+        </Style.Triggers>
+    </Style>
+    
+    <Style x:Key="NewValueDisplay" TargetType="{x:Type TextBlock}">
```

Comment:
I don't like this template. It works on a "magic string" value, which was originally the string we were going to show for null values.

Instead, I'd prefer the VM have a bool, and the textbox showing the null is shown/hidden based on that bool.